apps/src/templates/SpeedSlider.jsx

```diff
@@ -0,0 +1,233 @@
+var React = require('react');
+var color = require('../color');
+var Radium = require('radium');
+var dom = require('../dom');
+
+const style = {
+  sliderTrack: {
+    stroke: color.lightest_purple,
+    strokeWidth: 4,
+    strokeLinecap: "round",
+  },
+  sliderKnob: {
+    fill: "#ddd",
+    stroke: "#bbc",
+    strokeWidth: 1,
+    strokeLinejoin: "round",
+    ':hover': {
+      fill: "#eee"
```

Comment:
unfortunately, I can't get any of the colours we have to look good here. Leaving as a one-off for now.